prov/efa/src/rxr/rxr_rma.c

```diff
@@ -446,13 +446,17 @@ ssize_t rxr_rma_post_write(struct rxr_ep *ep, struct rxr_tx_entry *tx_entry)
 		else if (!rxr_peer_support_delivery_complete(peer))
 			return -FI_EOPNOTSUPP;
 
-		max_rtm_data_size = rxr_pkt_req_max_data_size(ep,
-							      tx_entry->addr,
-							      RXR_DC_EAGER_RTW_PKT);
+		max_rtm_data_size = rxr_pkt_calc_data_size(ep,
```

Comment:
take this opportunity to rename the variable `max_eager_rtw_data_size`, because it is not rtm.

Reply:
Where is this?